src/gui/gui2_canvas.cpp

```diff
@@ -1,13 +1,14 @@
 #include "gui2_canvas.h"
 #include "gui2_element.h"
 #include "theme.h"
+#include "preferenceManager.h"
```

Comment:
Overall, this the changes look decent. But I'm not sure about this specific part, not sure if I like it that the gui2_canvas depends on the preferencesManager. I think I will like it better if there is some class static or global that gets set from the "consumer" and keeps the gui2 code reasonably clean from dependencies.

Reply:
Yeah, I like it better this way. Not that the whole EE codebase is a showoff of great code design :P